### PR TITLE
fix: Update UI Handling for new TOS API

### PR DESCRIPTION
**Pull request by rstens**
*Issue:*

- bcgov/entity/issues/23180

*Description of changes:*
Changes to ```terms-of-service.vue``` and ```account.ts``` to account for TOS Changes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-136-f13la2m9.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-136-f13la2m9.web.app

**Comment by rstens** [comment]
BCREG0025 is set to false - ready for test.

File: strr-web/stores/account.ts [hunk 1]
```diff
@@ -71,17 +71,21 @@ export const useBcrosAccount = defineStore('bcros/account', () => {
...
   }
 
-  async function acceptTos (acceptance: boolean, versionId?: string): Promise<TermsOfServiceI | void> {
-    return await axiosInstance.patch<TermsOfServiceI>(`${strrApiURL}/account`,
+  async function acceptTos (acceptance: boolean, versionId: string): Promise<TermsOfServiceI | void> {
+    return await axiosInstance.patch<TermsOfServiceI>(`${strrApiURL}/users/tos`,
       {
-        acceptTermsAndConditions: acceptance,
-        termsVersion: versionId
+        istermsaccepted: acceptance,
```
Comment (dimak1) [inline strr-web/stores/account.ts:77]: No came case?

File: strr-web/stores/account.ts [hunk 2]
```diff
@@ -71,17 +71,21 @@ export const useBcrosAccount = defineStore('bcros/account', () => {
...
+        termsversion: versionId
       }
     )
       .then(() => {
         setAccountInfo()
           .then(() => {
-            navigateTo('/create-account')
+            if (acceptance) {
```
Comment (dimak1) [inline strr-web/stores/account.ts:84]: So we don't look at the response and redirect user regardless?

Comment (rstens) [reply strr-web/stores/account.ts:84]: No the navigateTo was removed.
```typescript
.then(() => {
        setAccountInfo()
          .then(() => {
            if (acceptance) {
              navigateTo('/create-account')
            } else {
              navigateTo('/')
            }
          })
      })
```

File: strr-web/stores/account.ts (hunk 1 above)
Comment (rstens) [reply strr-web/stores/account.ts:77]: Ask Lekshmi: get 500 error if I do not do lowercase

Comment (dimak1) [reply strr-web/stores/account.ts:77]: ok, I assume you and Lekshmi are ok with lowercase. Resolved the comment.

**Review by dimak1 — APPROVED** [review]

**Pull request merged**